stubs/Authlib/authlib/jose/drafts/_jwe_algorithms.pyi

```diff
@@ -0,0 +1,27 @@
+from _typeshed import Incomplete
+
+from authlib.jose.rfc7516 import JWEAlgorithmWithTagAwareKeyAgreement
+
+class ECDH1PUAlgorithm(JWEAlgorithmWithTagAwareKeyAgreement):
+    EXTRA_HEADERS: Incomplete
+    ALLOWED_KEY_CLS: Incomplete
+    name: str
```

Comment:
@JelleZijlstra 

I don't understand the stubtest errors in CI. for example, this case should be pretty straightforward.

```
error: authlib.jose.drafts._jwe_algorithms.ECDH1PUAlgorithm.name variable differs from runtime type None
Stub: in file stubs/Authlib/authlib/jose/drafts/_jwe_algorithms.pyi:8
builtins.str
Runtime:
None
```

the source is here: https://github.com/lepture/authlib/blob/master/authlib/jose/drafts/_jwe_algorithms.py#L19

under what circumstances would we exclude a file from the stubtest?

Reply:
I haven't looked through the full code but I suspect a base class has something like `name = None` in the class body, so that is what you see if you access the `.name` attribute on the class (which is all stubtest can do).

If that's the case, then this stub is right and it would be a good candidate for allowlisting the stubtest error. We wouldn't want to exclude the whole file, just this specific error. We can do that by adding `authlib.jose.drafts._jwe_algorithms.ECDH1PUAlgorithm.name` to a stubtest allowlist file.

Reply:
Ah, I see, I wasn't sure how stubtest works but now it makes sense. I'll look through the rest of these when I have time